src/encoded/renderers.py

```diff
@@ -306,15 +321,80 @@ def canonical_redirect(event):
     raise HTTPMovedPermanently(location=location, detail="Redirected from " + str(request.path_info))
 
 
```

Comment:
This code block about MIME is something that's been evolving in `cgap-portal`. I think it's good for CGAP and Fourfront to agree on a lot of this.